src/style.css

```diff
@@ -1 +1,67 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+.logo {
+  margin-right: auto;
+  margin-left: 50px;
+}
```

Comment:
let's remove these styles. here the problem is that you have the gap on the left of your heder set here and the gap on the right in a different place. this does not make much sense. why not simply set `padding: 0 50px` for the `.header`?

regarding the margin-right, you won't need that if you set `justify-content: space-between` for the `.header`.

do not overcomlicate your code. things that relate, should be in one place